x/derivatives/types/positions.go

```diff
@@ -84,12 +85,9 @@ func (m PerpetualFuturesPosition) NeedLiquidation(minMarginMaintenanceRate, curr
 func (m PerpetualFuturesPosition) EffectiveMargin(currentBaseUsdRate, currentQuoteUsdRate, currentMarginUsdRate sdk.Dec) sdk.Dec {
 	effectiveMargin := sdk.NewDecFromInt(m.RemainingMargin.Amount).Mul(currentBaseUsdRate.Quo(currentMarginUsdRate))
```

Comment:
effective margin fanc is exists in feat/run_msgreport_in_beginblock
plz use it

Reply:
where should I apply this fanc?
what is the original intention of this fanc?

Reply:
??
Sorry. I was confused.
Please ignore this Msg.